upgrade/scripts/k8s/upgrade_k8s_steps.sh

```diff
@@ -119,5 +119,18 @@ else
   fi
 fi
 
+if [[ -f "$DONE_DIR/rr_cs_rollout_restart.done" ]]; then
+  echo "INFO Rack Resiliency Critical services rollout restart completed, skipping."
+else
+  # For each CS in static ConfigMap, perform rollout restart of the critical service
+  python3 rr_critical_service_restart.py
+  if [[ $? -eq 0 ]]; then
+    echo "INFO Successfully completed critical services rollout restart"
+    touch "$DONE_DIR/rr_cs_rollout_restart.done"
+  else
+    echo "ERROR Either rollout restart of the critical services is not performed or failed"
```

Comment:
Doesn't this need to have an `exit 1` like the other clauses, so that this causes the failure to be visible to IUF?